.rubocop.yml

```diff
@@ -26,6 +26,7 @@ Layout/MultilineMethodCallIndentation:
 # https://github.com/mynewsdesk/mynewsdesk/pull/3205
 Metrics/LineLength:
   Max: 128
+  AllowURI: true
```

Comment:
Interesting group of settings. HEREDOC should also be good. But shouldn't it be true by default? Different versions perhaps?

Reply:
Yes according to docs it should be true. But I guess it isn't? Or the regexp is crap? I've wondered about this for years now...

Reply:
Maybe is not there yet for hound
http://help.houndci.com/en/articles/2461415-supported-linters
https://www.rubydoc.info/gems/rubocop/0.83.0/RuboCop/Cop/Layout/LineLength
(I'm having a hard time finding that option on rubydoc....)